easybuild/tools/module_generator.py

```diff
@@ -780,7 +780,7 @@ def getenv_cmd(self, envvar):
         """
         Return module-syntax specific code to get value of specific environment variable.
         """
-        return '$::env(%s)' % envvar
+        return '[if { [info exists ::env(%s)] } { concat $::env(%s) } ]' % (envvar, envvar)
```

Comment:
This implies the module file no longer errors if the variable is not in the environment